_bibliography/papers.bib

```diff
@@ -78,3 +78,15 @@ @article{qiao2019interlayer
   publisher={Elsevier},
   html={https://www.sciencedirect.com/science/article/pii/S0009261419301708}
 }
+@inproceedings{ribeiro-etal-2020-beyond,
+    title = {Beyond Accuracy: Behavioral Testing of NLP Models with CheckList},
+    author = {Ribeiro, Marco Tulio  and
+      Wu, Tongshuang  and
+      Guestrin, Carlos  and
+      Singh, Sameer},
+    booktitle = {Proceedings of the 58th Annual Meeting of the Association for Computational Linguistics},
+    year = {2020},
+    publisher = {Association for Computational Linguistics},
+    html = {https://www.aclweb.org/anthology/2020.acl-main.442},
+    pages = {4902--4912}
+}
```

Comment:
This is a good format for the reference (and it is exactly same as in Overleaf, in the `custom.bib` file) but your references should **not** be added to this file. Instead, you should create a new file named `2021-08-01-BERTdepression.bib`, add all the necessary references there, and put it to the folder `assests/bibliography`